powersimdata/input/tests/test_transform_profile.py

```diff
@@ -191,56 +203,63 @@ def base_grid():
 
 
 @pytest.fixture(scope="module")
-def base_hydro():
+def raw_hydro():
     input_data = InputData()
     profile_info = {
-        "interconnect": "_".join(interconnect),
-        "base_hydro": param["_".join(interconnect)]["hydro"],
+        "grid_model": "usa_tamu",
+        "base_hydro": param["hydro"],
     }
     profile = input_data.get_data(profile_info, "hydro")
     return profile_info, profile
 
 
 @pytest.fixture(scope="module")
-def base_wind():
+def raw_wind():
     input_data = InputData()
     profile_info = {
-        "interconnect": "_".join(interconnect),
-        "base_wind": param["_".join(interconnect)]["wind"],
+        "grid_model": "usa_tamu",
+        "base_wind": param["wind"],
     }
     profile = input_data.get_data(profile_info, "wind")
     return profile_info, profile
 
 
 @pytest.fixture(scope="module")
-def base_solar():
+def raw_solar():
     input_data = InputData()
     profile_info = {
-        "interconnect": "_".join(interconnect),
-        "base_solar": param["_".join(interconnect)]["solar"],
+        "grid_model": "usa_tamu",
+        "base_solar": param["solar"],
     }
     profile = input_data.get_data(profile_info, "solar")
     return profile_info, profile
 
 
-@pytest.fixture(scope="module")
-def base_demand():
-    input_data = InputData()
-    profile_info = {
-        "interconnect": "_".join(interconnect),
-        "base_demand": param["_".join(interconnect)]["demand"],
-    }
-    profile = input_data.get_data(profile_info, "demand")
-    return profile_info, profile
+# @pytest.fixture(scope="module")
+# def raw_demand():
+#     input_data = InputData()
+#     profile_info = {
+#         "interconnect": "_".join(interconnect),
+#         "grid_model": "usa_tamu",
+#         "base_demand": param["demand"],
+#     }
+#     profile = input_data.get_data(profile_info, "demand")
+#     return profile_info, profile
```

Comment:
Is there a reason we keep this here?

Reply:
Nope. I just commented it before moving it but did not erase it afterwards. Thanks.

Reply:
Done